cluster-sync/sync.sh

```diff
@@ -27,7 +27,6 @@ PULL_POLICY=$(getTestPullPolicy)
 # build and push it's localhost, but for manifests, we sneak in a change to point a registry container on the
 # kubernetes cluster.  So, we introduced this MANIFEST_REGISTRY variable specifically to deal with that and not
 # have to refactor/rewrite any of the code that works currently.
-MANIFEST_REGISTRY=$DOCKER_PREFIX
```

Comment:
why?

This seems to be breaking manifest generation for me (not using my DOCKER_PREFIX)

Reply:
I accidentally deleted that I think, I will restore